DP3TApp/Logic/Tracing/TracingManager.swift

```diff
@@ -277,3 +284,37 @@ extension TracingManager: DP3TBackgroundHandler {
         }
     }
 #endif
+
+extension TracingManager: ActivityDelegate {
+    func syncCompleted(totalRequest: Int, errors: [DP3TNetworkingError]) {
+        let encoding = Array("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+        let numberOfSuccess = totalRequest - errors.count
+        let numberOfErrors = errors.count
+        var payload = String(encoding[0])
```

Comment:
I think this is different from the Android solution: https://github.com/DP-3T/dp3t-sdk-android/pull/145/files#diff-c3708376fd980de4633e9ea97e7b3f32R239